### PR TITLE
Add a hire link along with pages for authors.

### PR DESCRIPTION
**Pull request by NikhilVerma**
And some minor tweaks to CSS and categories.

### CODE REVIEW

File: _posts/2015-02-09-code-signing-and-distributing-swift.markdown [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: post
 title:  Code signing and distributing - Swift
 author: Miguel Angel Quinones
 date:   2015-02-09
-categories: ios swift CI
```
Comment (netproteus) [inline _posts/2015-02-09-code-signing-and-distributing-swift.markdown]: Why did you remove these categories?

Comment (NikhilVerma) [inline _posts/2015-02-09-code-signing-and-distributing-swift.markdown]: We want to keep the category list simple. People are using categories as hash tags. So for now me and Kelly have discussed to keep the category list to what it is now and only add new categories when needed.
